seleniumByTechnocredits/src/seleniumPractise/ankush/RegistrationFormValidation.java

```diff
@@ -0,0 +1,177 @@
+		e6.click();
+
+		if (RegistrationFormValidation.alertHandling(driver).equals("Please enter your current organization")) {
+			System.out.println("Company Name Validation Successful");
+			driver.findElement(By.id("organization")).sendKeys("Test Organization");
+			System.out.println("Current Organization Entered");
+
+		} else
+			System.out.println("Current Organization Validation Failed");
+
+		// User Name Name Field Validation
+		e5.click();
+
+		if (RegistrationFormValidation.alertHandling(driver).equals("Username is mandatory field.")) {
+			System.out.println("User Name Validation Successful");
+			driver.findElement(By.id("usernameReg")).sendKeys("12345");
+			e5.click();
+			if (RegistrationFormValidation.alertHandling(driver)
+					.equals("Username length should be greater then 5 characters.")) {
+				driver.findElement(By.id("usernameReg")).clear();
+				driver.findElement(By.id("usernameReg")).sendKeys("TestUser");
+				System.out.println("User Name Entered");
+
+			} else
+				System.out.println("User Name Validation Failed");
+
+		}
+
+		// Password Field Validation
+		e5.click();
+
+		if (RegistrationFormValidation.alertHandling(driver).equals("password is mandatory field.")) {
+			driver.findElement(By.id("passwordReg")).sendKeys("1234");
+			e5.click();
+			if (RegistrationFormValidation.alertHandling(driver)
+					.equals("password length should be greater then 5 characters.")) {
+				driver.findElement(By.id("passwordReg")).clear();
+				driver.findElement(By.id("passwordReg")).sendKeys("TestPassword");
+				System.out.println("Password Entered");
+
+			} else
+				System.out.println("Password Validation Failed");
+
+		}
+
+		// Reenter Password Field Validation
+		e5.click();
+
+		if (RegistrationFormValidation.alertHandling(driver).equals("please reenter password")) {
+			driver.findElement(By.id("repasswordReg")).sendKeys("1234");
+			e5.click();
+			if (RegistrationFormValidation.alertHandling(driver).equals("retype password donot match.")) {
+				driver.findElement(By.id("repasswordReg")).clear();
+				driver.findElement(By.id("repasswordReg")).sendKeys("TestPassword");
+				System.out.println("ReEntered Password");
+
+			} else
+				System.out.println("Re Entered Validation Failed");
+
+		}
+
+		// Privacy Policy Field Validation
+
+		e5.click();
+		if (RegistrationFormValidation.alertHandling(driver)
+				.equals("Please agree to terms of service and privacy policy")) {
+			driver.findElement(By.id("signupAgreement")).click();
+			e5.click();
+		} else
+			System.out.println("Privacy Policy Validation Failed");
+
+		if (RegistrationFormValidation.alertHandling(driver).equals("Success")) {
+			System.out.println("All Validation are passed Successfully");
+		} else
+			System.out.println("Validations Fails");
+
+	}
+
+	public static void main(String[] args) throws InterruptedException {
+		WebDriver driver = Init.initChromeDriver();
+		RegistrationFormValidation.registrationFormChecks(driver);
```

Comment:
For all the else part in code we will get exception alert unhandled .Please handle the same.
Format the code .